lib/web_ui/lib/src/engine/fonts.dart

```diff
@@ -2,14 +2,23 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
+import 'dart:async';
 import 'dart:typed_data';
 
 import 'assets.dart';
 
 abstract class FontCollection {
   Future<void> loadFontFromList(Uint8List list, {String? fontFamily});
```

Comment:
Document this as well. Also make sure that `registerDownloadedFonts` doesn't need to be called after this, since it is used by the framework.